i18n/extract.py

```diff
@@ -65,6 +65,22 @@ def add_args(self):
         Adds arguments
         """
         self.parser.description = __doc__
+        self.parser.add_argument(
+            '--merge-js',
+            action='store_true',
+            help='Merge djangojs.po with django.po using msgcat'
+        )
+        self.parser.add_argument(
+            '--no-partial',
+            action='store_true',
+            help=(
+                'Renames (django-partial.po) and (djangojs-partial.po) to (django.po) and (djangojs.po) respectively. '
+                'This will also remove (django-saved.po) and (djangojs-saved.po) that are used to restore original '
+                '(django.po) and (djangojs.po) after extract is completed. '
+                'This is helpful when this `extract` command output is needed to be used immediately '
+                'with no segment/merge workflow in small repositories.'
```

Comment:
```suggestion
                'Renames (django-partial.po) and (djangojs-partial.po) to (django.po) and (djangojs.po) respectively. '
                'This will also remove (django-saved.po) and (djangojs-saved.po) that are used to restore the original '
                '(django.po) and (djangojs.po) files after the extraction is completed. '
                'This is helpful when running the `extract` command in small repositories '
                'with no segment/merge workflow.'
```

Reply:
Done